io/net/multicast.py.data/multicast.yaml

```diff
@@ -1,4 +1,4 @@
 Parameters:
-    peer_ip: ""
+    peer_ip: "10.68.68.41"
     user_name: "root"
-    interface: ""
+    interface: "enp96s0f0"
```

Comment:
Please do not add a default parameter that are needed to run the test, since a user can always run it without editing and might face effects which they might not like.